src/Catalyst.Cli.Tests/UnitTests/Commands/Response/VerifyMessageResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void VerifyMessageResponse_Can_Get_Output()
             //Arrange
             var verifyMessageResponse = new VerifyMessageResponse();
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var messageVerifyCommand = new MessageVerifyCommand(commandContext, Substitute.For<ILogger>());
+            new MessageVerifyCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'MessageVerifyCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)